scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -383,19 +383,41 @@ class FossId(
         return results
     }
 
+    private suspend fun List<Scan>.findLatestPendingOrFinishedScan(url: String, revision: String? = null): Scan? {
+        return sortedByDescending { scan -> scan.id }.find { scan ->
+            // The scans in the server contain the url with the credentials so we have to remove it for the
+            // comparison. If we don't, the scans won't be matched if the password changes!
+            val urlWithoutCredentials = scan.gitRepoUrl?.replaceCredentialsInUri()
+            if (urlWithoutCredentials == url && (revision == null || (scan.gitBranch == revision))) {
+                val scanCode = requireNotNull(scan.code) {
+                    "FossId returned a null scancode for an existing scan"
```

Comment:
Missing '.' at the end of the sentence.

Reply:
Are you sure about this ? This is not a log message rather an exception message....

Reply:
I think, this is a convention in the complete ORT codebase.

Reply:
After a quick search this is in fact inconsistent in ORT. However I will fix it for you :)